metadata-ingestion/src/datahub/ingestion/source/state/stale_entity_removal_handler.py

```diff
@@ -34,10 +34,11 @@ class StatefulStaleMetadataRemovalConfig(StatefulIngestionConfig):
         description="Soft-deletes the entities present in the last successful run but missing in the current run with stateful_ingestion enabled.",
     )
     fail_safe_threshold: float = pydantic.Field(
-        default=20.0,
+        default=100.0,
         description="Prevents large amount of soft deletes & the state from committing from accidental changes to the source configuration if the relative change percent in entities compared to the previous state is above the 'fail_safe_threshold'.",
-        le=100.0,  # mypy does not work with pydantic.confloat. This is the recommended work-around.
+        le=100.0,
         ge=0.0,
+        hidden_from_schema=True,
```

Comment:
what does this do?

Reply:
That hides the option from the docs site